### DISCUSSION TITLE
Second computer doesn't connect anymore

### DISCUSSION THREAD

**Discussion by la-mikkamakka** [Chia-Network/chia-blockchain#5224] [Support]
Hello, My second computer doesn't connect anymore to the network, it just stalls on connecting to wallet. I have tried to connect via command prompt and it says I can connect only one computer to the port 8444. But I have tries a laptop for testing and it is connecting without any issues.

So I have tried to move the HDD to the H1 harvester, added directory to the plots and now it cannot recognize the plots moved from #2 to #1 harvester.

What should i do? How can I merge plots from 2 different harvesters?
How can I het #2 harvester to connect to network again? I was working for about 4 days. 
I have port forwarded in router firewall from any source to any destination too.

**Comment by la-mikkamakka** [comment]
update, I have tried to add #2 wallet to #1 harvester and after a restart the plots were recognized and added to my total plot count. Unfortunately the wallet syncing is still not moving, got stuck at 3231 and hasn't moved for days. Not sure why that is.

#2 harvester seems to a loss at this point, reinstalling software, rebooting computer has not helped at all.
We need fixing for that. If there is anything out for multi computer harvesting from the same IP, would you please direct me to it?

Also, my temp folders are holding the plot remains that got cut off when I restarted the program. Why isn't the program picking up the same tread? Recognizing that it was started on this computer? there should be a resume option, like  a download manager that keeps track of files, folders and download status. Can anyone look into that option? It would save a lot of tears when computers crash.

Thanks

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
